src/templates/next/components/shared/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,58 @@
+import { BiLeftArrowAlt, BiRightArrowAlt } from "react-icons/bi"
+import type { PaginationProps } from "../../../types/Pagination"
+
+export const Pagination = ({
+  totalItems,
+  itemsPerPage,
```

Comment:
note that we need some level of check for `itemsPerPage` - doesn't have to be here as it'll be used both on FE + end user site. 

maybe we can just default this to 10?

Reply:
Hmm not doing this. I feel that the `itemsPerPage` is a required prop that should be provided by the caller. Providing defaults, understandably sane ones, may add a layer of complexity as it makes the caller provide one thing and the component providing another, which is then combined to implement the functionality of the component.

Overall I think forcing the caller to control everything would be safer in the longer term. Just in case, I added a minimum of 1 to the `itemsPerPage` prop below.